image_test.go

```diff
@@ -707,6 +708,7 @@ func TestBuildImageParameters(t *testing.T) {
 		"cpuperiod":  {"100000"},
 		"cpusetcpus": {"0-3"},
 		"ulimits":    {"[{\"Name\":\"nofile\",\"Soft\":100,\"Hard\":200}]"},
+		"buildargs":  {"[{\"Name\":\"SOME_VAR\",\"Value\":\"some_value\"}]"},
```

Comment:
Can you change both lines to use strings with `` please? Then we don't need to escape the " inside the string:

```
 		"ulimits":    {`[{"Name":"nofile","Soft":100,"Hard":200}]`},
		"buildargs":  {`[{"Name":"SOME_VAR","Value":"some_value"}]`},
```

I know that you didn't add the ulimits line, but can you fix this one as well?

Thanks!